ibis/backends/tests/test_dot_sql.py

```diff
@@ -353,7 +353,8 @@ def test_unnamed_columns(con):
     assert len(names) == 2
 
     assert names[0]
-    assert names[1] == "col42"
```

Comment:
We should actually quote though, because that was my intention. I opened https://github.com/ibis-project/ibis/pull/10032 for that.